lib/pure/parsesql.nim

```diff
@@ -501,6 +501,7 @@ type
     nkColumnReference,
     nkReferences,
     nkDefault,
+    nkAutoIncrement,
```

Comment:
> remain old problem:

if there are remaining problems, then either remove `Fix #15977` from PR description (which i just added) or add a future work section in PR description